intellij/.ideavimrc

```diff
@@ -0,0 +1,62 @@
+"" Source your .vimrc
+"source ~/.vimrc
+scriptencoding utf-8
+set encoding=utf-8
+syntax on
+
+set exrc
+set noerrorbells
+
+set shiftwidth=4
+set tabstop=4
+set softtabstop=4
+set expandtab
+set autoindent
+set smartindent
+set nostartofline
+
+set number relativenumber
+set nowrap
+set colorcolumn=80
+set signcolumn=yes
+set scrolloff=8
+
+set wildmenu
+set showcmd
+
+set hlsearch
+set smartcase
+set ignorecase
+set incsearch
+
+set backspace=indent,eol,start
+set ruler
+set laststatus=2
+set confirm
+set visualbell
+set t_vb=
+
+set ttimeoutlen=100
+set updatetime=50
+imap jj <Esc>
```

Comment:
Nit: I personally find `jk` to be a more satisfying way of escaping insert mode, personally.
```
imap jk <Esc>
```

Reply:
Addressed in 67113d2d89f386a8162eebe487f986c8d55c669a